firmware/projects/FrontController/inc/simp_vd_interface.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "app.h"
+
+struct VdInput {
+    float driver_torque_request;
+    float brake_pedal_postion;
+    float steering_angle;
+    float wheel_speed_lr;
+    float wheel_speed_rr;
+    float wheel_speed_lf;
+    float wheel_speed_rf;
+    bool tv_enable;
+};
+
+struct VdOutput {
+    float lm_torque_limit_positive;
+    float rm_torque_limit_positive;
+    float lm_torque_limit_negative;
+    float rm_torque_limit_negative;
```

Comment:
@BlakeFreer What data type are these outputs? The Simulink model prefixes them VD_T but I assumed them to be floats.
![image](https://github.com/user-attachments/assets/bcf2eca9-1f5a-4c3e-a7fe-dd4dc4e6101a)

Reply:
Yeah use floats